src/services/cohortCreation/index.ts

```diff
@@ -16,10 +23,25 @@ const getPatients = memoize(
     if (!query) return []
     const response = await apiFhir.get<FHIR_API_Response<IPatient>>(query)
     const patients = getApiResponseResources(response)
-    return patients ? patients.map((patient) => patient.id) : []
+    return patients?.map((patient) => patient.id) ?? []
   }
 )
 
+const getPatientsFromDocuments = memoize(
```

Comment:
Could we do a single query with an `_include` or a `_has` instead of that?

Reply:
Can I with the operator? I tried without success

Reply:
Oh, you're right, maybe we need to implement something 🤔
I'll try to see if we can do that